src/client/pydaos/__init__.py

```diff
@@ -0,0 +1,68 @@
+# (C) Copyright  2019 Intel Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+# The Government's rights to use, modify, reproduce, release, perform, display,
+# or disclose this software are subject to the terms of the Apache License as
+# provided in Contract No. B609815.
+# Any reproduction of computer software, computer software documentation, or
+# portions thereof marked with this legend must also reproduce the markings.
+"""
+PyDAOS Module allowing global access to the DAOS containers and objects.
+"""
+
+import sys
+import atexit
+
+DAOS_MAGIC = 0x7A89
+
+# pylint: disable=no-member
```

Comment:
Where does the warning occur that is being ignored by this statement?  The position of this is somewhat worrying as it could be masking real issues unintentionally.

Reply:
fixed too.